data/apps.ts

```diff
@@ -35,3 +35,9 @@ export const apps = [
     logo: 'lenstube.svg'
   }
 ];
+
+export const lensterCommunity = {
+  id: 'lenster community',
+  name: 'Lenster Community',
+  logo: 'lenster.svg'
+};
```

Comment:
**suggestion:** can we move this to `apps` object?

Reply:
yes, we can:

- so, in Feed, I would filter it out
- and posts by the "lenster community" would look like this:

before vs. after
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/10981565/183653612-980de339-aebb-4d9f-9d04-929d45f5a86d.png">

--

<img width="1309" alt="image" src="https://user-images.githubusercontent.com/10981565/183653748-c2c6c163-fdcb-4465-8ee2-3fd1dd4d7075.png">

Reply:
p.s. I specifically didn't want to add it to apps at first, cause checked that this will cause this change

also, is the same icon + name "Lenster Community" ok?

we could do the name "Lenster", so it would show same as lenster

Reply:
But I'm not seeing `lensterCommunity` used anywhere 🤔 

Even if we add to `apps` it will show `Posted via Lenster Community` but we don't want to show it and it doesn't makes sense, it's just a community creation 🤔

Reply:
Also we have lot of sources not listed in the file, it might be from small apps but still we want to show everything except the community creations

Reply:
I left a message in the commit message and in the description that this lensterCommunity is only added for visibility that it exists - yes, I didn't use it :)

--

that was my other comment - "filter out" from apps

```
apps.map(app => app.id).filter(id => id !== 'lenster community')
```

--

so the next option would be to filter on the front-end and in bad cases:
- we query with a limit of 10 items
<img width="303" alt="image" src="https://user-images.githubusercontent.com/10981565/183659107-cee851d2-9687-411f-8b97-4eb207414798.png">
- we get 9 communities
- we show only 1 post in top collected

I haven't found any other way in API to filter, for example, by attributes

Reply:
Yup we are blocked by the backend, you can see more here

https://github.com/aave/lens-api-examples/issues/48

Reply:
that totally would be my solution :)

so, wait for now?
I can clean PR and leave it in a pending state

Reply:
so, if Explore for Top Collected needs to have small "unregistered" apps to be present - I think we should wait 😬

I am totally against filtering on FE with the case that I described - you didn't propose, I didn't propose, but to be clear about it 😅